frb_rust/build.rs

```diff
@@ -0,0 +1,7 @@
+use build_target::{self, Family};
+
+fn main() {
+    if let Ok(Family::Wasm) = build_target::target_family() {
+        println!("cargo:rustc-cfg=wasm");
+    }
```

Comment:
Oh cargo is so silly... It knows it is wasm while do not tell rustc about it?

Reply:
This one is a leftover from past commits, I'll clean it up.